operations/phlare/helm/phlare/templates/deployments-statefulsets.yaml

```
@@ -65,6 +65,10 @@ spec:
               value: {{ $value | quote }}
            {{- end }}
           {{- end }}
+          {{- with .Values.extraEnvFrom }}
```

We allow a couple of overrides to be happening and hence you need to access the values like this:

```suggestion
          {{- with .$values.extraEnvFrom }}
```